pylint/checkers/imports.py

```diff
@@ -190,7 +190,7 @@ def _make_graph(filename: str, dep_info: Dict[str, List[str]], sect: VNode, gtyp
     report's section
     """
     outputfile = _dependencies_graph(filename, dep_info)
-    sect.append(Paragraph(f"{gtype}imports graph has been written to {outputfile}"))
+    sect.append(Paragraph((f"{gtype}imports graph has been written to {outputfile}",)))
```

Comment:
In order to fix:
```
pylint/checkers/imports.py:193: error: Argument 1 to "Paragraph" has incompatible type "str"; expected "Union[List[Text], Tuple[str, ...]]"
Found 1 error in 1 file (checked 197 source files)
```

Reply:
Based on `BaseLayout` I think those expected value are incorrect.